testing/mgba/APKBUILD

```diff
@@ -0,0 +1,85 @@
+# Contributor: Alexander Edland <alpine@ocv.me>
+# Maintainer: Alexander Edland <alpine@ocv.me>
+pkgname=mgba
+pkgver=0.6.3
+pkgrel=0
+pkgdesc="Game Boy Advance Emulator"
+url="https://github.com/mgba-emu/mgba"
+arch="all"
+license="MPL-2.0"
+depends=""
+depends_dev=""
+makedepends="cmake qt5-qtmultimedia-dev qt5-qtbase-dev qt5-qttools-dev
+	sdl-dev libzip-dev zlib-dev libpng-dev ffmpeg-dev imagemagick-dev
+	libepoxy-dev cmocka-dev libedit-dev py3-virtualenv python3-dev"
+checkdepends="py3-cffi py3-cached-property"
+install=""
+subpackages="$pkgname-doc $pkgname-py:_py"
+source="mgba-$pkgver.tar.gz::https://github.com/mgba-emu/mgba/archive/$pkgver.tar.gz
+	http://gbdev.gg8.se/files/roms/blargg-gb-tests/cpu_instrs.zip
+	no-rpath.patch
+	python3.patch
+	mgba-qt.desktop"
+builddir="$srcdir/mgba-$pkgver"
+
+build() {
+	cd "$builddir"
+	
+	# declaration mismatch in include/mgba/core/thread.h
+	export CPPFLAGS="$CPPFLAGS -fno-strict-aliasing"
+	
+	mkdir build
+	cd build
+	cmake \
+		-DCMAKE_INSTALL_PREFIX:PATH=/usr \
+		-DCMAKE_INSTALL_LIBDIR:PATH=/usr/lib \
+		-DBUILD_TEST:BOOL=ON \
+		-DBUILD_SUITE:BOOL=ON \
+		-DBUILD_PYTHON:BOOL=ON \
+		..
+	make
+}
+
+check() {
+	cd "$builddir/build"
+	./mgba-suite
+	
+	# test python bindings by executing an open source ROM
+	# for DMG-01 (OG gameboy) and exporting a png screenshot
+	PYTHONPATH="$PWD/$(echo python/lib.linux-x86_64-3.*)" \
+	LD_LIBRARY_PATH="$builddir/build" \
+	ZIPFILE="$srcdir/cpu_instrs.zip" \
+	/bin/sh "$startdir/test-py.sh"
+}
+
+package() {
+	cd "$builddir/build"
+	make install DESTDIR="$pkgdir"
+	
+	# delete test files
+	rm "$pkgdir/usr/bin/mgba-fuzz"
+	rm "$pkgdir/usr/bin/tbl-fuzz"
+	
+	# add desktop file
+	local _apps="$pkgdir/usr/share/applications"
```

Comment:
Local variables don't need leading underscores (also goes for the ones in `_py`.